frontend/src/metabase/visualizations/click-actions/modes/EmbeddingSdkMode.ts

```diff
@@ -0,0 +1,34 @@
+import type { QueryClickActionsMode } from "../../types";
+// import { ColumnFormattingAction } from "../actions/ColumnFormattingAction";
+import { DashboardClickAction } from "../actions/DashboardClickAction";
+import { HideColumnAction } from "../actions/HideColumnAction";
+import { NativeQueryClickFallback } from "../actions/NativeQueryClickFallback";
+
+export const EmbeddingSdkMode: QueryClickActionsMode = {
+  name: "default",
```

Comment:
is this the name of the mode? I saw we're having duplicate names right now. There are 4 modes in this directory, 2 have `default` as a name, and the others have `public`.

I'd expect this to be named like `embedding-sdk` or something.